apps/web/src/app/(common-page)/lab/lab-item-card.tsx

```diff
@@ -157,6 +158,11 @@ export const LabItemCard: React.FC<LabItemCardProps> = (props) => {
                 <IconBrandGithub /> Git repository
               </Li>
             )}
+            {props.docs && (
+              <Li to={props.docs}>
+                <IconForms /> Documentation
```

Comment:
Ik wil hier `IconFileTypeDoc` gebruiken ipv `IconForms` maar als ik die toevoeg aan de `import {...} from '@tabler/icons-react';` dan krijg ik in vsc de onderstaande foutmelding. Geen idee hoe dat te fixen.

![image](https://github.com/MinBZK/regels.overheid.nl/assets/25812095/2d80f99d-0f0a-4cc3-b7e2-30b5977b1176)